### PR TITLE
Slvs 1070

### PR DESCRIPTION
**Pull request by tvsonar**

### CODE REVIEW

File: src/SonarAnalyzer.Common/Rules/Utilities/TokenTypeAnalyzerBase.cs [hunk 1]
```diff
@@ -194,6 +202,15 @@ private void ClassifyIdentifier(SyntaxToken token, ISymbol symbol)
                 }
             }
 
+            private int myVar;
+
+            public int MyProperty
```
Comment (SonarTech) [inline src/SonarAnalyzer.Common/Rules/Utilities/TokenTypeAnalyzerBase.cs]: MINOR Make this an auto-implemented property and remove its backing field. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS2292)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue
* MINOR 1 minor

Watch the comments in this conversation to review them.

File: src/SonarAnalyzer.Common/Rules/Utilities/TokenTypeAnalyzerBase.cs [hunk 2]
```diff
@@ -181,6 +181,14 @@ private void ClassifyIdentifier(SyntaxToken token, ISymbol symbol)
                     return;
                 }
 
+                if (token.ToString() == "value" &&
```
Comment (Evangelink) [inline src/SonarAnalyzer.Common/Rules/Utilities/TokenTypeAnalyzerBase.cs]: Is it worth putting a const here for "value"?

Comment (tvsonar) [inline src/SonarAnalyzer.Common/Rules/Utilities/TokenTypeAnalyzerBase.cs]: I don't think so. String literals will be interned, so they will automatically reference the same constant. See here
* https://msdn.microsoft.com/en-us/library/system.string.intern(v=vs.110).aspx#Anchor_2
* https://blogs.msdn.microsoft.com/ericlippert/2009/09/28/string-interning-and-string-empty/

**Pull request merged**